diff --git a/src/server.rs b/src/server.rs
index d53931b..ce9f080 100644
--- a/src/server.rs
+++ b/src/server.rs
@@ -1,7 +1,7 @@
 use std::{
     env,
     io::{BufRead, BufReader},
-    path::PathBuf,
+    path::Path,
     process::{self, Child, Command},
     thread,
 };
@@ -16,8 +16,7 @@ pub struct Server {
 
 impl Server {
     pub fn new() -> Self {
-        let server_path = env::var("SERVER_PATH").expect("Failed to read SERVER_PATH env");
-        let file = PathBuf::from(&server_path);
+        let file = Path::new("@serverjs@").to_path_buf();
 
         Self {
             process: None,
